### ISSUE TITLE
The Model seem to have a problem to try to fit with the face on the Deepfacelive , but I tried other models it does look good , might be the problem on the training state , can some one help me how to avoid it ? 

### ISSUE THREAD

**Issue by Ben4434** [iperov/DeepFaceLab#5691]
I'm trying to train my own models to learn how it work , and encounter some problem with the result and I can't figure it out where the problem is , is it the Xseg mask ? 
![Screenshot 2023-06-28 191401](https://github.com/iperov/DeepFaceLab/assets/87603383/b0179c45-6f05-4c31-9e92-6771e67af326)
and here some of the setting I'm running on 
================== Model Summary ===================
==                                                ==
==            Model name: new_SAEHD               ==
==                                                ==
==     Current iteration: 3207409                 ==
==                                                ==
==---------------- Model Options -----------------==
==                                                ==
==            resolution: 224                     ==
==             face_type: wf                      ==
==     models_opt_on_gpu: True                    ==
==                 archi: liae-udt                ==
==               ae_dims: 512                     ==
==                e_dims: 64                      ==
==                d_dims: 64                      ==
==           d_mask_dims: 32                      ==
==       masked_training: True                    ==
==       eyes_mouth_prio: True                    ==
==           uniform_yaw: True                    ==
==         blur_out_mask: True                    ==
==             adabelief: True                    ==
==            lr_dropout: y                       ==
==           random_warp: False                   ==
==      random_hsv_power: 0.1                     ==
==       true_face_power: 0.0                     ==
==      face_style_power: 0.0                     ==
==        bg_style_power: 0.0                     ==
==               ct_mode: none                    ==
==              clipgrad: False                   ==
==              pretrain: False                   ==
==       autobackup_hour: 0                       ==
== write_preview_history: False                   ==
==           target_iter: 0                       ==
==       random_src_flip: False                   ==
==       random_dst_flip: True                    ==
==            batch_size: 32                      ==
==             gan_power: 0.1                     ==
==        gan_patch_size: 28                      ==
==              gan_dims: 32                      ==
==                                                ==
==------------------ Running On ------------------==
==                                                ==
==          Device index: 0                       ==
==                  Name: NVIDIA GeForce RTX 4090 ==
==                  VRAM: 21.70GB                 ==
==                                                ==
====================================================

**Comment by kingace4q** [comment]
Did you get a solution to this?
